pkg/asset/internal/templates.go

```diff
@@ -225,6 +225,48 @@ spec:
     type: RollingUpdate
 `)
 
+var BootstrapFluentdTemplate = []byte(`apiVersion: v1
+kind: Pod
+metadata:
+  name: bootstrap-fluentd
+  namespace: kube-system
+spec:
+  containers:
+  - name: bootstrap-fluentd
+    image: {{ .Images.Fluentd}}
+    env:
+    - name: FLUENTD_OPT
+      value: "--no-supervisor -q"
+    resources:
+      limits:
+        memory: 300Mi
+      requests:
+        cpu: 100m
+        memory: 200Mi
+    volumeMounts:
+    - name: varlog
+      mountPath: /var/log
+      readOnly: false
+    - name: varlibdockercontainers
+      mountPath: /var/lib/docker/containers
+      readOnly: true
+    - name: libsystemddir
+      mountPath: /host/lib
+      readOnly: true
+  hostNetwork: true  
+  volumes:
+  - name: varlog
+    hostPath:
+      path: /var/log
+  - name: varlibdockercontainers
+    hostPath:
+      path: /var/lib/docker/containers
+  - name: libsystemddir
+    hostPath:
+      path: /usr/lib64
```

Comment:
The fluentd team provides an image that doesn't have this requirement:

https://github.com/fluent/fluentd-kubernetes-daemonset/blob/master/fluentd-daemonset-elasticsearch.yaml